test/detail/BitStreamTests.cpp

```diff
@@ -117,7 +117,7 @@ TEST_CASE("BitStream") {
            const auto n = *gen::inRange(0, 64);
            const bool sign = (x & (1LL << (n - 1LL))) != 0;
```

Comment:
I'm actually surprised ASAN isn't throwing warnings on this line; If `n==0`, this shift will decompose to `(1LL << -1LL)`.
It might be clearer to set `n = *gen::inRange(1, 64)` in this and the previous test, and add new property that specifically checks the behavior of `stream.next<int64_t>(0)`?